### PR TITLE
feat(types): add/update contrib types

### PR DESCRIPTION
**Pull request by kentcdodds**
To "upgrade" to the latest specification.

I'm not sure whether to consider this a breaking change...

### CODE REVIEW

**Comment by kentcdodds** [comment]
Rats, I might have to actually pull this down sometime...

File: README.md [hunk 1]
```diff
@@ -88,8 +92,9 @@ These are the keys you can specify:
...
...ontributors-cli/commits?author=jfmengels) [📖](https://github.com/jfmengels/all-contributors-cli/commits?author=jfmengels) [⚠️](https://github.com/jfmengels/all-contributors-cli/commits?author=jfmengels) | [<img src="https://avatars.githubusercontent.com/u/1500684?v=3" width="100px;"/><br /><sub>Kent C. Dodds</sub>](http://kentcdodds.com/)<br />[📖](https://github.com/jfmengels/all-contributors-cli/commits?author=kentcdodds) | [<img src="https://avatars.githubusercontent.com/u/14871650?v=3" width="100px;"/><br /><sub>João Guimarães</sub>](https://github.com/jccguimaraes)<br />[💻](https://github.com/jfmengels/all-contributors-cli/commits?author=jccguimaraes) | [<img src="https://avatars.githubusercontent.com/u/1282980?v=3" width="100px;"/><br /><sub>Ben Briggs</sub>](http://beneb.info)<br />[💻](https://github.com/jfmengels/all-contributors-cli/commits?author=ben-eb) | [<img src="https://avatars.githubusercontent.com/u/22768990?v=3" width="100px;"/><br /><sub>Itai Steinherz</sub>](https://github.com/itaisteinherz)<br />[📖](https://github.com/jfmengels/all-contributors-cli/commits?author=itaisteinherz) [💻](https://github.com/jfmengels/all-contributors-cli/commits?author=itaisteinherz) |
```
Comment (itaisteinherz) [inline README.md]: I think the contributors list got duplicated because of 92e2aa29e9d786b496348d79867e6aa837e8e045.

**Comment by kentcdodds** [comment]
I was hoping that I wouldn't have to actually pull it down (I'm so impressed with how much GitHub's web client is capable of these days!) but I got the formatting wrong on the table :face_palm: so I've pulled it down and regenerated things :) Should be good now I think. Will let @jfmengels merge and release if he thinks it's ready.

**Comment by jfmengels** [comment]
Thanks to you both!
@kentcdodds You made this a breaking change in the spec right? Do you think this should be a major release then? I think probably not, but I'm down with whatever.

By the way, I just gave you publishing rights, I trust your judgement in handling this tool, you use it more than I do :smile:

**Comment by kentcdodds** [comment]
Do you mind if I setup semantic-release on it?

**Comment by jfmengels** [comment]
No I don't, but I have never learnt how to use it (or rather, what the commit messages should look like). I guess it's time to do so :D

**Comment by kentcdodds** [comment]
Luckily with GitHub's new "Rebase and Merge" functionality, you can change the commit message of PRs when you merge them right in GitHub's UI which is pretty amazingly awesome. I'll get things setup. Thanks!

**Pull request merged**